src/Apps/NewWorksFromGalleriesYouFollow/NewWorksFromGalleriesYouFollowApp.tsx

```diff
@@ -0,0 +1,140 @@
+import { Box, Button, Spacer, Text } from "@artsy/palette"
+
+import { MetaTags } from "Components/MetaTags"
+import { FC, useState } from "react"
+import {
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import { NewWorksFromGalleriesYouFollowApp_me$data } from "__generated__/NewWorksFromGalleriesYouFollowApp_me.graphql"
+import ArtworkGrid from "Components/ArtworkGrid/ArtworkGrid"
+import { ContextModule, AuthContextModule } from "@artsy/cohesion"
+import { LogInPrompt } from "Apps/Components/LogInPrompt"
+
+export interface NewWorksFromGalleriesYouFollowAppProps {
+  me: NewWorksFromGalleriesYouFollowApp_me$data
+  relay: RelayPaginationProp
+}
+
+const NewWorksFromGalleriesYouFollowApp: FC<NewWorksFromGalleriesYouFollowAppProps> = ({
+  me,
+  relay,
+}) => {
+  const [isLoading, setLoading] = useState(false)
+
+  const { newWorksFromGalleriesYouFollowConnection } = me
+
+  const handleLoadMore = () => {
+    if (!relay.hasMore() || relay.isLoading()) return
+
+    setLoading(true)
+
+    relay.loadMore(25, err => {
+      if (err) console.error(err)
+      setLoading(false)
+    })
+  }
+
+  return (
+    <>
+      <MetaTags
+        title="New Works from Galleries You Follow | Artsy"
+        pathname="/new-works-from-galleries-you-follow"
+      />
+
+      <Spacer y={4} />
+
+      <Text variant="xl">New Works from Galleries You Follow</Text>
+
+      <Spacer y={4} />
+
+      <LogInPrompt
```

Comment:
This component shouldn't have an external margin.

Reply:
Not sure what you mean by external margin 🤔 

I've copied the code from https://www.artsy.net/new-for-you which shows the component the same way.

Reply:
It's not part of this PR but this `LogInPrompt` component has a size 4 margin hanging off the bottom of it. This makes it both confusing to use and brittle. https://github.com/artsy/force/blob/main/docs/best_practices.md#avoid-external-margins